lib/setupmanagers/hckclient.rb

```diff
@@ -114,9 +119,16 @@ def copy_dvl
       end
     end
 
+    def machine_in_working_pool?
```

Comment:
This method should no longer be necessary. You can just call `pool` and check the returned value.